api/api-doc/storage_service.json

```diff
@@ -1228,7 +1228,7 @@
          "operations":[
             {
                "method":"POST",
-               "summary":"Removes token (and all data associated with enpoint that had it) from the ring",
+               "summary":"Removes a node from the cluster. Replicated data that logically belonged to this node is remapped and restored onto the remaining nodes, if possible.",
```

Comment:
s/remapped and restored onto/redistributed among/

Why the `if possible`? Is there a situation under this is dong possible? A failure is always implicit I think, no need to include it in the message.

Reply:
I think we can be in a degraded situation where we cannot satisfy the replication factor if removing a node when RF == number of nodes.

Reply:
done in v3